includes/wikia/Wikia.php

```diff
@@ -312,15 +310,15 @@ static public function fixDomainName( $name, $language = false, $type = false )
 					case "answers":
 						$domains = self::getAnswersDomains();
 						if ( $language && isset($domains[$language]) && !empty($domains[$language]) ) {
-							$name =  sprintf("%s.%s.%s", $name, $domains[$language], $wgWikiaBaseDomain);
+							$name =  sprintf("%s.%s.%s", $name, $domains[$language], "wikia.com");
 							$allowLang = false;
 						} else {
-							$name =  sprintf("%s.%s.%s", $name, $domains["default"], $wgWikiaBaseDomain);
+							$name =  sprintf("%s.%s.%s", $name, $domains["default"], "wikia.com");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "default" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)